clang/lib/CAS/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(LLVM_LINK_COMPONENTS
+  Support
```

Comment:
Shouldn't this depend on LLVM CAS as well? I think this would fail to link dynamically.

Reply:
Fixed in e9567d33bfadb11309d954266554aca4d05977e3.